### ISSUE TITLE
How to uninstall local installation

### ISSUE THREAD

**Issue by githlp** [LibrePhotos/librephotos-linux#51]
As is installed librephotos for local install and now try to make a docker installation: How do i uninstall the local installation ?

I think it installed about ~ 3000 packages to my server, is there any clean way to undo it ?

**Comment by afeuerstein** [comment]
Yes, please. I am currently running the installer, watching loads of packages being installed, in fright of the uninstall process should I decide that I do not like LibrePhotos.
